src/v2/Apps/Consign/Routes/NewMarketingLanding/Components/FAQ.tsx

```diff
@@ -0,0 +1,138 @@
+          learn more.
+        </TextItem>
+      ),
+    },
+    {
+      label: "I am an artist. Can I sell my art with Artsy?",
+      value: (
+        <TextItem>
+          <p>
+            Currently, Artsy does not directly partner with artists outside of
+            our approved artists list. Instead, we offer partnerships for
+            galleries, museums, and other institutions.
+          </p>
+          <p>
+            If you are represented by a gallery, you can encourage them to
+            partner with us by getting in touch at{" "}
+            <a href="mailto:partners@artsy.net">partners@artsy.net</a>.
+          </p>
+          <p>
+            If you’re looking for gallery representation, you might be
+            interested in reading Artsy Editorial’s{" "}
+            <a
+              href="/article/artsy-editorial-artists-gallery-representation"
+              target="_blank"
+            >
+              Gallery Representation Guide
+            </a>{" "}
+            and{" "}
+            <a href="/series/working-artists-guide" target="_blank">
+              Working Artist’s Guide
+            </a>
+            .
+          </p>
+        </TextItem>
+      ),
+    },
+    {
+      label: "I am a gallerist. How can I partner with Artsy?",
+      value: (
+        <TextItem>
+          Visit{" "}
+          <a href="https://partners.artsy.net" target="_blank">
+            partners.artsy.net
+          </a>{" "}
+          to apply to become a gallery partner and learn more about our
+          marketplace plans.
+        </TextItem>
+      ),
+    },
+    {
+      label: "Which items do you accept?",
+      value: (
+        <TextItem>
+          We are currently accepting unique and limited-edition works of art by
+          modern, contemporary, and emerging artists who have collector demand
+          on Artsy. Our in-house experts cover Post-War and Contemporary Art;
+          Prints and Multiples; and Street Art and Photographs.
+        </TextItem>
+      ),
+    },
+    {
+      label: "How do you price artworks for sale?",
+      value: (
+        <TextItem>
+          To assess a work’s value, our team uses publicly available auction
+          results, as well as Artsy’s own database of primary market sales. We
+          also take into account market trends (including both an individual
+          artist’s market and macro trends in the art market), artwork
+          condition, and sales channel, as well as other factors that may impact
+          a work’s value, such as medium, provenance, subject matter, or period.
+        </TextItem>
+      ),
+    },
+  ]
+
+  return (
+    <>
+      <Text mb={4} variant="lg" id="jump--faq">
+        Frequently Asked Questions
+      </Text>
+      <Join separator={<Spacer mt={4} />}>
+        {FAQList.map(({ label, value }) => (
+          <Expandable onFocus={() => trackClickedFAQ(label)} label={label}>
```

Comment:
In both cases. Main benefit of using `onFocus` here it's to prevent a series of clicks on the same item and track only one click event.

Reply:
Also for now we track the analytics event only on the thank you page(submission flow)

Reply:
This may be a question for @audrey-thenot, but if we fire the same event for both "expand" and "collapse," then won't it look like the FAQs are getting more engagement (potentially double) than they really are?